### DISCUSSION TITLE
Using FFT with two channels.

### DISCUSSION THREAD

**Discussion by OSBringer** [pschatzmann/arduino-audio-tools#762] [Q&A] [answered]
Hello everyone,
I recently started using FFT with INMP441 microphone through 

```
AudioRealFFT fft;
I2SStream i2sStream;
StreamCopy copier(fft, i2sStream);
```

Since I am farely new to this topic, I was just wondering how does the library use behaves when using 2 channels.
Does it average the values combining the channels to mono or does it calculate each FFT value for both of the channels?

Cheers

**Comment by pschatzmann** [comment, marked as answer]
You can define the channel_used in the configuration to define which channel should be used to perform the fft on.
If you want to combine the channels you can use [this](https://github.com/pschatzmann/arduino-audio-tools/wiki/Converting-the-Data-Format)
